tests/functional/test_cross_entropy.py

```diff
@@ -115,16 +115,15 @@ def _reverse_kl_forward_backward_torch(target: torch.Tensor, logits: torch.Tenso
 @pytest.mark.parametrize("loss_masking", [False, True])
 @pytest.mark.parametrize("target_format", (TargetFormat.logits,))
 def test_reverse_kl(loss_masking, target_format):
-    logits, target, loss_mask = _get_cross_entropy_inputs(10000, loss_masking, target_format)
-    out_ref, grad_ref = _reverse_kl_forward_backward_torch(logits, target, loss_mask)
+    logits, target, loss_mask = _get_cross_entropy_inputs(1000, loss_masking, target_format)
+    out_ref, grad_ref = _reverse_kl_forward_backward_torch(target, logits, loss_mask)
```

Comment:
Switch to `logits, target` to match `reverse_kl_forward_backward` and clear up the confusion?